SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -152,6 +152,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Wait Time Before Processing Received RPC With Unresolved Refs"))
 	float QueuedIncomingRPCWaitTime;
 
+	/** Seconds to wait before attempting to reprocess queued incoming RPCs */
+	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Wait Time Before Attempting To Reprocess Queued Incoming RPCs"))
+	float QueuedIncomingRPCFlushTime;
```

Comment:
`QueuedIncomingRPCRetryTime` ?

Reply:
Useful to call out the special behaviour if this is zero, like in the comment below.